src/OrleansLoggerProvider/LegacyOrleansLoggerProvider.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.Extensions.Logging;
+using Orleans.Runtime;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+
+namespace Orleans.Extensions.Logging.Legacy
+{
+    /// <summary>
+    /// Provides an ILoggerProvider, whose implementation try to preserve orleans legacy logging features and abstraction
+    /// OrleansLoggerProvider creates one ILogger implementation, which supports orleans legacy logging features, including <see cref="ILogConsumer"/>, 
+    /// <see cref="ICloseableLogConsumer">, <see cref="IFlushableLogConsumer">, <see cref="Severity">. 
+    /// LegacyOrleansLoggerProvider also supports configuration on those legacy features.
+    /// </summary>
+    [Obsolete("Use Microsoft.Extensions.Logging built-in logger providers")]
+    public class LegacyOrleansLoggerProvider : ILoggerProvider
+    {
+        /// <summary>
+        /// Default Severity for all loggers
+        /// </summary>
+        public static Severity DefaultSeverity = Severity.Info;
```

Comment:
Why should we make this public?

Reply:
so that user know what is the default Severity?